bot/utils/scheduling.py

```diff
@@ -13,9 +14,10 @@ class Scheduler(metaclass=CogABCMeta):
     """Task scheduler."""
 
     def __init__(self):
+        # Keep track of the child cog's name so the logs are clear.
+        self.cog_name = self.__class__.__name__
 
-        self.cog_name = self.__class__.__name__  # keep track of the child cog's name so the logs are clear.
-        self.scheduled_tasks: Dict[str, asyncio.Task] = {}
+        self._scheduled_tasks: Dict[str, asyncio.Task] = {}
```

Comment:
I like this change; it signals that we really should be using the Scheduler API instead of manipulating the mapping of scheduled tasks directly (which bypasses the checks, callbacks, and logging steps done below).